sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1958,6 +1973,15 @@ class MultiTaskLasso(MultiTaskElasticNet):
         number of iterations run by the coordinate descent solver to reach
         the specified tolerance.
 
+    dual_gap_ : ndarray of shape (n_alphas,)
+        The dual gaps at the end of the optimization for each alpha
+
+    sparse_coef_ : sparse matrix of shape (n_tasks, n_features)
+        ``sparse_coef_`` is a readonly property derived from ``coef_``
+
+    eps_: float, default=1e-4
+        same as tol
```

Comment:
Please revert these unrelated changes
```
git checkout master --  sklearn/linear_model/_coordinate_descent.py 
```

If you want to add them, better to make a new PR.